src/main/java/topaz/task/Task.java

```diff
@@ -0,0 +1,78 @@
+package topaz.task;
+
+import java.util.ArrayList;
+
+import topaz.exception.InvalidStateException;
+/**
+ * Represents a task with a description and a completion status.
+ * A {@link Task} object has a description and a boolean flag indicating whether the task is done.
+ */
+

```

Comment:
Should the blank line be removed to follow the coding standard?

Reply:
Already removed, thanks!